packages/@react-spectrum/tree/docs/TreeView.mdx

```diff
@@ -0,0 +1,512 @@
+          <Text>Document 1</Text>
+          <FileTxt />
+        </TreeViewItem>
+        <TreeViewItem id="document-2" textValue="Document 2">
+          <Text>Document 2</Text>
+          <FileTxt />
+        </TreeViewItem>
+      </TreeViewItem>
+      <TreeViewItem id="photos" textValue="Photos">
+        <Text>Photos</Text>
+        <Folder />
+        <TreeViewItem id="image-1" textValue="Image 1">
+          <Text>Image 1</Text>
+          <Image />
+        </TreeViewItem>
+        <TreeViewItem id="image-2" textValue="Image 2">
+          <Text>Image 2</Text>
+          <Image />
+        </TreeViewItem>
+        <TreeViewItem id="image-3" textValue="Image 3">
+          <Text>Image 3</Text>
+          <Image />
+        </TreeViewItem>
+      </TreeViewItem>
+    </TreeView>
+  );
+}
+```
+
+### Single selection
+
+To limit users to selecting only a single item at a time, `selectionMode` can be set to `single`.
+
+```tsx example
+// Using the same tree as above
+<ExampleTree selectionMode="single" />
+```
+
+### Disallow empty selection
+
+TreeView also supports a `disallowEmptySelection` prop which forces the user to have at least one item in the TreeView selected at all times.
+In this mode, if a single item is selected and the user presses it, it will not be deselected.
+
+```tsx example
+// Using the same tree as above
+<ExampleTree disallowEmptySelection />
+```
+
+### Disabled items
+
+You can disable specific items by providing an array of keys to TreeView via the `disabledKeys` prop. This will prevent items from being selectable as shown in the example below.
+
+```tsx example
+// Using the same tree as above
+<ExampleTree selectionMode="multiple" disabledKeys={['document-1', 'document-2']} />
+```
+
+### Highlight selection
+
+By default, TreeView uses the checkbox selection style, which includes a checkbox in each item for selection. When the selectionStyle prop is set to `"highlight"`, the checkboxes are hidden,
+and the selected items are displayed with a highlighted background instead.
+
+In addition to changing the appearance, the selection behavior also changes depending on the `selectionStyle` prop. In the default checkbox selection style, clicking, tapping, or pressing
+the <Keyboard>Space</Keyboard> or <Keyboard>Enter</Keyboard> keys toggles selection for the focused item. Using the arrow keys moves focus but does not change selection.
+
+In the highlight selection style, however, clicking a item with the mouse _replaces_ the selection with only that item. Using the arrow keys moves both focus and selection. To select multiple items,
+modifier keys such as <Keyboard>Ctrl</Keyboard>, <Keyboard>Cmd</Keyboard>, and <Keyboard>Shift</Keyboard> can be used. On touch screen devices, selection always behaves as toggle since modifier
+keys may not be available.
+
+These selection styles implement the behaviors defined in [Aria Practices](https://www.w3.org/WAI/ARIA/apg/patterns/listbox/#keyboardinteraction).
```

Comment:
listbox? or do we implement https://www.w3.org/WAI/ARIA/apg/patterns/treeview/ ?